rosdep/base.yaml

```diff
@@ -4670,6 +4670,9 @@ mosquitto-dev:
   debian: [libmosquitto-dev]
   fedora: [mosquitto-devel]
   ubuntu: [libmosquitto-dev]
+mosquittopp-dev:
+  debian: [libmosquittopp-dev]
+  ubuntu: [libmosquittopp-dev]
```

Comment:
For development packages in particular, I think that we should be aiming for ease-of-use, rather than as-small-as-possible.  Thus, I think that it would be better to fold these into the `mosquitto-dev` key above.  @ros/rosdeputies , thoughts?